src/kayenta/stages/kayentaStage/kayentaStage.html

```diff
@@ -186,7 +186,7 @@
     </kayenta-stage-config-section>
   </for-analysis-type>
 
-  <kayenta-stage-config-section title="Baseline + Canary Server Groups">
+  <kayenta-stage-config-section title="Baseline + Canary Pair">
```

Comment:
We have been using 'pair' and 'server groups' semi-interchangeably, this is my attempt at picking one. I also went singular because we only allow a maximum of one right now.

Reply:
Thanks, I like this better.